Update Russian version replacingmergetree.md. Added is_deleted parameter description

grant88 opened the pull request:
Update Russian version replacingmergetree.md. Added is_deleted parameter **description**

Added is_deleted parameter description for russian documentaion version


### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

grant88 committed:
Update Russian version replacingmergetree.md. dded is_deleted parameter description

Added is_deleted parameter description for russian documentaion version

den-crane commented on docs/ru/engines/table-engines/mergetree-family/replacingmergetree.md:
```diff
@@ -86,6 +86,53 @@ SELECT * FROM mySecondReplacingMT FINAL;
+:::note
+`is_deleted` может быть использован, если `ver` используется.
+
+Строка удаляется при использовании инструкции `OPTIMIZE ... FINAL CLEANUP` или `OPTIMIZE ... FINAL`, или в случае, если параметр `clean_deleted_rows` установлен в значение `Always`.
```
или если есть новые версии строки https://fiddle.clickhouse.com/e6f31b17-76cc-4cfc-9e05-d1b8e40741d3

grant88 replied:
Добавил

den-crane commented:
вообще как-то сформулировать что делать CLEANUP и использовать clean_deleted_rows=always очень нерекомендуется , потому что оно может привести к неверным результатам, удаленные строки например воскреснут.

https://github.com/ClickHouse/ClickHouse/issues/49685#issuecomment-1546261014
https://github.com/ClickHouse/ClickHouse/issues/50346

grant88 committed:
Update replacingmergetree.md cleanup+clean_deleted_rows warning

Добавил формулировку, что CLEANUP + clean_deleted_rows=always - не следует использовать

grant88 commented:
Добавил предостережение, что не рекомендуется использовать CLEANUP и clean_deleted_rows=always

grant88 committed:
Update replacingmergetree.md remove simultaneously word

correct: remove simultaneously word of using FINAL CLEANUP and  clean_deleted_rows

grant88 commented on docs/ru/engines/table-engines/mergetree-family/replacingmergetree.md:
```diff
@@ -95,10 +95,16 @@ SELECT * FROM mySecondReplacingMT FINAL;
+    - параметр движка `clean_deleted_rows` установлен в значение `Always` (по умолчанию - `Never`)
+    - есть новые версии строки
+
+Не рекомендуется одновременно выполнять `CLEANUP` и использовать параметр движка `clean_deleted_rows`, это может привести к неожиданным результатам, например удаленные строки могут внось появиться.
```
Поправил

robot-ch-test-poll commented:
<!-- automatic status comment for PR #55584 from grant88/ClickHouse:patch-3 -->
*This is an automated comment for commit 52480fe092b11b4e634951c3813d4519ba21e805 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/55584/52480fe092b11b4e634951c3813d4519ba21e805/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6508306609">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/55584/52480fe092b11b4e634951c3813d4519ba21e805/ci_running.html">❌ failure</a></td></tr>
<tbody>
</table>
